src/hotspot/share/gc/g1/heapRegion.cpp

```diff
@@ -615,6 +616,10 @@ class VerifyRemSetClosure : public G1VerificationClosure {
           if (!_failures) {
             log.error("----------");
           }
+          {
+            LogStream ls(log.error());
+            to->rem_set()->print_info(&ls, p);
+          }
```

Comment:
Instead of adding this scope, can't we just create the `LogStream` here and the re-use it below?